commands/categories/bits.md

```diff
@@ -6,7 +6,7 @@
     computed: {
       commands() {
         return pages
-          .filter(p => p.path.includes('/book/commands/'))
+          .filter(p => p.path.includes('/commands/commands/'))
```

Comment:
There are 28 `/commands/commands/` need to be changed to `/commands/docs/`

Reply:
Weird, seems like that change got lost somewhere, fixed now.